src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/PublicationTests.cs

```diff
@@ -0,0 +1,160 @@
+                {
+                    new()
+                    {
+                        Id = release2000Id,
+                        Version = 0,
+                        ReleaseName = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        Published = DateTime.UtcNow.AddDays(-2),
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+
+                    },
+                    new()
+                    {
+                        Id = release2001Id,
+                        Version = 0,
+                        ReleaseName = "2001",
+                        Published = DateTime.UtcNow.AddDays(-1),
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                    },
+                    new()
+                    {
+                        Id = release2002Id,
+                        Version = 0,
+                        ReleaseName = "2002",
+                        Published = null,
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        ApprovalStatus = ReleaseApprovalStatus.Draft,
+                    },
+                },
+            };
+
+            var result = publication.LatestPublishedRelease();
+            Assert.Equal(release2001Id, result.Id);
+        }
+
+        [Fact]
+        public void LatestRelease()
+        {
+            var release2000Id = Guid.NewGuid();
+            var release2000AmendmentId = Guid.NewGuid();
+            var release2001Id = Guid.NewGuid();
+            var release2001AmendmentId = Guid.NewGuid();
+            var publication = new Publication
+            {
+                Releases = new List<Release>
+                {
+                    new()
+                    {
+                        Id = release2000Id,
+                        Version = 0,
+                        ReleaseName = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        Published = DateTime.UtcNow.AddDays(-2),
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+
+                    },
+                    new()
+                    {
+                        Id = release2000AmendmentId,
+                        Version = 1,
+                        PreviousVersionId = release2000Id,
+                        ReleaseName = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        Published = DateTime.UtcNow.AddDays(-1),
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                    },
+                    new()
+                    {
+                        Id = release2001Id,
+                        Version = 0,
+                        ReleaseName = "2001",
+                        Published = DateTime.UtcNow.AddDays(-1),
```

Comment:
Same comment about Published date/time as above. If they were all published on the same day we should expect the same result, but the data is setting up days -1 and -2 which made me think it's influencing the order in some way.